app/src/main/java/com/muharuto/introduceclubactivities/AddClubSummaryFragment.kt

```diff
@@ -0,0 +1,21 @@
+package com.muharuto.introduceclubactivities
+
+import android.os.Bundle
+import android.view.View
+import androidx.fragment.app.Fragment
+import com.muharuto.introduceclubactivities.databinding.FragementAddClubSummaryBinding
+
+class AddClubSummaryFragment : Fragment(R.layout.fragement_add_club_summary) {
+    private var fragmentAddClubSummaryBinding: FragementAddClubSummaryBinding? = null
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        val binding = FragementAddClubSummaryBinding.bind(view)
+        fragmentAddClubSummaryBinding = binding
+    }
+
+    override fun onDestroyView() {
+        fragmentAddClubSummaryBinding = null
+        super.onDestroyView()
+    }
```

Comment:
```suggestion
    override fun onDestroyView() {
        super.onDestroyView()
        fragmentAddClubSummaryBinding = null
    }
```

少し見ずらいので親クラスのメソッドを呼び出す右場合は関数のすぐ下に書きましょう👀